lib/samples/style_graphics_with_renderer/style_graphics_with_renderer.dart

```diff
@@ -0,0 +1,291 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import 'dart:math';
+
+import 'package:arcgis_maps/arcgis_maps.dart';
+import 'package:arcgis_maps_sdk_flutter_samples/utils/sample_state_support.dart';
+import 'package:flutter/material.dart';
+
+class StyleGraphicsWithRenderer extends StatefulWidget {
+  const StyleGraphicsWithRenderer({super.key});
+
+  @override
+  State<StyleGraphicsWithRenderer> createState() =>
+      _StyleGraphicsWithRendererState();
+}
+
+class _StyleGraphicsWithRendererState extends State<StyleGraphicsWithRenderer>
+    with SampleStateSupport {
+  // Create a controller for the map view.
+  final _mapViewController = ArcGISMapView.createController();
+
+  // A flag for when the map view is ready and controls can be used.
+  var _ready = false;
```

Comment:
`_ready` isn't providing any benefit for this sample, so we can strip it and the progress indicator

Reply:
Done removed `_ready`